tests/unit/test-cmd/test.run.js

```diff
@@ -325,5 +326,101 @@ describe('run', () => {
            sourceDir: '/fake/source/dir',
          },
        );
-     });
-});
+     }
+  );
+
+  describe('profile-create-new option', () => {
+    beforeEach(() => {
+      sinon.stub(fs, 'mkdir');
+    });
+
+    afterEach(() => {
+      fs.mkdir.restore();
+    });
+
+    it('creates dir when firefox profile doesn\'t exist',
+       async () => {
+         sinon.stub(fs, 'existsSync').returns(false);
+         const firefoxProfile = '/pretend/path/to/Firefox/profile';
+         const cmd = prepareRun();
+
+         await cmd.run({
+           firefoxProfile,
+           profileCreateIfMissing: true,
+         });
+
+         sinon.assert.calledWith(fs.mkdir, firefoxProfile);
+
+         fs.existsSync.restore();
+       }
+    );
+
+    it('creates dir when chromium profile doesn\'t exist',
+       async () => {
+         sinon.stub(fs, 'existsSync').returns(false);
+         const chromiumProfile = '/pretend/path/to/Chromium/profile';
+         const cmd = prepareRun();
+
+         await cmd.run({
+           chromiumProfile,
+           target: 'chromium',
+           profileCreateIfMissing: true,
+         });
+
+         sinon.assert.calledWith(fs.mkdir, chromiumProfile);
+
+         fs.existsSync.restore();
```

Comment:
I think that we should restore fs.existsSync in the afterEach (and stub it in beforeEach, eventually changing what it does return in the particular test if necessary) as for the other stub, in case one test case fails and `fs.existsSync.restore()` isn't reached.